src/create.rs

```diff
@@ -409,13 +428,58 @@ pub struct Execution {
     locale: Option<PathBuf>,
     name: Option<String>,
     no_build: bool,
+    wix_toolset: WixToolset,
+    wix_toolset_upgrade: WixToolsetUpgrade,
     install: bool,
     output: Option<String>,
     package: Option<String>,
     target: Option<String>,
     version: Option<String>,
 }
 
+/// Enumeration of wix-toolset options
+/// 
+/// This controls which wix build binaries are used by cargo-wix
+#[derive(ValueEnum, Copy, Clone, Debug)]
+pub enum WixToolset {
+    /// The default wix toolset uses "candle.exe" and "light.exe" to build the installer
+    Default,
```

Comment:
I would rename `Default` to `Legacy` or `CandleAndLight` or change both variants to be `V3` and `V4` (instead of `Modern`) to be more explicit/descriptive.

I would then implement the `Default` trait on the `WixToolset` type that would return the `Legacy` or `V3` variant.

Reply:
I think I'll go for `Legacy`, the reason I wanted to avoid using V3 and V4 was because I didn't want to give an impression that if you're using WiX5 you need to use `--toolset V5`, because for example in Wix5 they don't even use a different schema because it's backwards compatible (weird to even call it a major version but  🤷‍♂️), so the only time this is relevant is if you're going between v3 <-> v4.